components/tag/src/tag-list/tag-list.html

```diff
@@ -6,7 +6,7 @@
       *ngIf="_showAllTags"
     ></ng-template>
   </div>
-  <div class="dt-tag-list-more-btn" *ngIf="!_isOneLine && !_showAllTags">
+  <div class="dt-tag-list-more-btn" *ngIf="_displayMoreBtn">
```

Comment:
call the function here directly 
```suggestion
  <div class="dt-tag-list-more-btn" *ngIf="_showDisplayMoreButton()">
```